x-pack/solutions/security/plugins/security_solution/public/common/hooks/use_navigate_to_alerts_page_with_filters.ts

```diff
@@ -16,14 +16,18 @@ import { URL_PARAM_KEY } from './use_url_state';
 export const useNavigateToAlertsPageWithFilters = () => {
   const { navigateTo } = useNavigation();
 
-  return (filterItems: FilterControlConfig | FilterControlConfig[], openInNewTab = false) => {
+  return (
+    filterItems: FilterControlConfig | FilterControlConfig[],
+    openInNewTab = false,
+    timerange?: string
```

Comment:
Currently, no-one (except the usage in the file `alerts_findings_details_table.tsx` above) is using the `openInNewTab` property. It seems to me that the new `timerange` property is fully related to the fact that we want to open in a new tab. If `openInNewTab`is false, the user is just navigating to a different page in Kibana within the same browser tab, and the KQL bar and its date time picker remain the same (meaning the date and time are persisted).

I've been trying to find a good way to combine `openInNewTab` and `timerange` somehow together?, but I'm not liking any of the solutions I can come up with.

I'm ok keeping things as you have right now, but if you have an idea let me know!

Reply:
would you be ok adding these prop documentations?

```typescript
    /**
     * Pass one or more filter control configurations to be applied to the alerts page filters
     */
    filterItems: FilterControlConfig | FilterControlConfig[],
    /**
     * If true, the alerts page will be opened in a new tab
     */
    openInNewTab = false,
    /**
     * Allows to customize the timerange url parameter. Should only be used in combination with the openInNewTab=true parameter
     */
    timerange?: string
```